test/unit/services/test_item.py

```diff
@@ -98,6 +98,17 @@
     "created_time": MODEL_MIXINS_FIXED_DATETIME_NOW,
     "modified_time": MODEL_MIXINS_FIXED_DATETIME_NOW,
 }
+
+# Load in the usage statuses data
+with open("./data/usage_statuses.json", "r", encoding="utf-8") as file:
+    usage_statuses_data = json.load(file)
+    USAGE_STATUSES = [
+        {"_id": usage_status["_id"]["$oid"], "value": usage_status["value"]} for usage_status in usage_statuses_data
+    ]
+USAGE_STATUSES_LIST = [
+    UsageStatusOut(id=usage_status["_id"], value=usage_status["value"]) for usage_status in USAGE_STATUSES
+]
+
```

Comment:
Unit tests shouldn't interact with the file system as that makes tests less isolated and more susceptible to external factors. Could you please hardcode a couple of usage statuses and use those when mocking instead?